src/main/resources/native/windows/launcher.c

```diff
@@ -27,7 +27,8 @@
  */
 #include <stdio.h>
 
-extern void *IsolateEnterStub__JavaMainWrapper__run__5087f5482cc9a6abc971913ece43acb471d2631b__a61fe6c26e84dd4037e4629852b5488bfcc16e7e();
+extern int *run_main(int argc, char* argv[]);
```

Comment:
The actual invocation of the method at line 37 should be changed to `run_main`.

Reply:
right, fixed.